tools/wake-format/actions.h

```diff
@@ -121,7 +143,29 @@ struct TokenReplaceAction {
     auto it = traits.find(node);
     if (it != traits.end()) {
       for (auto n : it->second.before_bound) {
-        builder.append(n.fragment().segment().str());
+        if (n.id() == TOKEN_COMMENT) {
```

Comment:
I'm going to pull an Ashley and ask if we can factor out the repeated code between the repeated code here. I'm happy to not do it though as well.